Clinical-Genomics/cgweb
# Fix issue where samples data_analysis becoming MIP for OP-added Balsamic samples

This PR fixes the issue with samples data_analysis becoming MIP for OP-added Balsamic samples

How to test:
1. install on clinical-db stage
1. create new Balsamic or Balsamic + MIP order
1. add new sample in OP to the order (as compared to importing an Orderform)
1. submit

Expected outcome: 
In the database and LIMS there should be a new sample with the data_analys == 'Balsamic' (or 'MIP + Balsamic' depending on your order type)
